lib/core/src/lib/card-view/components/card-view-dateitem/card-view-dateitem.component.scss

```diff
@@ -26,8 +26,8 @@
         float: right;
     }
 
-    .adf-dateitem-chip-list-container.adf-property-field {
-        margin-bottom: -7px !important;
+    &-card-view-dateitem .adf-dateitem-chip-list-container.adf-property-field {
```

Comment:
Why do we need to add this selector here?

Reply:
This extra selector is not needed. Removed.

Tested the UI, works as expected with/without !important